codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -341,4 +395,12 @@ public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         super.handlerRemoved(ctx);
         cleanup();
     }
+
+    public final int blockSize() {
+        return blockSize;
+    }
+
+    public final int currentBlockLength() {
+        return currentBlockLength;
```

Comment:
Hold off on reviewing ... I'm running into a problem with enforcing the max buffer size (from the buffer returned from `allocateBuffer()` and the reuse of the member field `buffer` across multiple `encode()` invocations.